Taarafo.Core/Services/Foundations/Events/EventService.cs

```diff
@@ -27,6 +28,14 @@ public partial class EventService : IEventService
             this.loggingBroker = loggingBroker;
         }
 
+        public ValueTask<Event> AddEventAsync(Event @event) =>
+        TryCatch(async () =>
```

Comment:
Looks good.  Can you just indent the TryCatch

Reply:
done